list/README.md

```diff
@@ -19,14 +19,206 @@ but already use ready-made mechanisms to handle lists in Python.
 
 ## Merge lists
 
+Python can merge lists with each other in a variety of ways, in this section we will go through the most popular merged lists and which theses are offered by the latest version of Python.
```

Comment:
Python can merge lists with each other in a variety of ways. In this section we will go through the most popular ways to 
 merge lists and which are offered by the latest version of Python.